addons/mail/static/src/components/message/message.js

```diff
@@ -15,6 +15,7 @@ const useUpdate = require('mail/static/src/component_hooks/use_update/use_update
 
 const { _lt } = require('web.core');
 const { getLangDatetimeFormat } = require('web.time');
+const { format } = require('web.field_utils');
```

Comment:
guideline: Import should be sorted alphabetically based on string inside require (`f` before `t` in this case).